manifests/charts/istio-control/istio-discovery/values.yaml

```diff
@@ -41,6 +41,8 @@ pilot:
   podAnnotations: {}
   serviceAnnotations: {}
 
+  topologySpreadConstraints: []
```

Comment:
@yitingdc This should be added in the operator API as well to pass the test.
https://github.com/istio/istio/tree/master/operator/pkg/apis/istio/v1alpha1

Reply:
thanks very much，done